rust_src/.gitignore

```diff
@@ -1 +1,2 @@
 target
+generated/
```

Comment:
I think `generated` is better than `generated/`.

Reply:
Better how? `generated` is less specific and would also match files that end up named `generated`.

Reply:
It makes sense.